tendrl/commons/flows/import_cluster/gluster_help.py

```diff
@@ -53,32 +58,27 @@ def import_gluster(parameters):
     try:
         out, err = runner.run()
         if out['rc'] != 0:
+            err_msg = "Could not install tendrl-gluster-integration " \
+                "on Node %s. Error: %s" % (NS.node_context.fqdn, out['msg'])
             logger.log(
                 "error",
                 NS.publisher_id,
-                {
-                    "message": "Could not install "
-                    "tendrl-gluster-integration on Node %s"
-                    "Error: %s" %
-                    (NS.node_context.fqdn, out['msg'])
-                },
+                {"message": err_msg},
                 job_id=parameters['job_id'],
                 flow_id=parameters['flow_id'],
             )
-            return False
+            return False, err_msg
     except ansible_module_runner.AnsibleExecutableGenerationFailed:
+        err_msg = "Error: Could not install tendrl-gluster-integration " \
```

Comment:
Please add to msg something like , cluster management requires service tendrl-gluster-integration

Reply:
Done. Will change the message.